submissions/mayowaOgunronbi/JavaScript/conditionals/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Conditionals/Loops</title>
+    <script src="script.js"></script>
```

Comment:
<script> should be added at the end of the </body> tag not inside the <head> section